js-intro.js

```diff
@@ -40,23 +50,72 @@ if (imLost || imFrustrated) {
 // 5. Update the imLost and imFrustated variables above the statement to the values required to log "Keep coding!"
 
 // 6a. Recall the mantra variable from the first question. Write the code that logs each letter of mantra using a for loop.
-
+// for(let i=0; i<mantra.length;i++){
+//     console.log(mantra.charAt(i))
+// }
 // Stretch: Using a while loop.
-
+// var i = 0
+// while(i<mantra.length){
+//     console.log(mantra.charAt(i))
+//     i++
+// }
 // Super Stretch: Using forEach().
+var mantraArray = mantra.split("")
+var i = 0
+//why cant i declare i inside the function? because each time it will reset it? is there a way around that?
+mantraArray.forEach(function(){
+    let arrayVal = mantraArray[i]
+    i++
+    console.log(arrayVal)
+})
 
```

Comment:
ForEach is a function that acts on every item in an array. You don't need to set the initial i value or the iteration. It's implied. 
var array1 = mantra.split("")
array1.forEach(function(element) {
  console.log(element)
})